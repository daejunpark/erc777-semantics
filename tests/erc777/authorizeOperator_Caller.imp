test.preamble();

// Testing authorizeOperator(Operator) when Operator = caller
// must revert

test.authorizeOperator(7);
