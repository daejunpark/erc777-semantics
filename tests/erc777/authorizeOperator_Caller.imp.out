7 is authorizing 7 as an operator
REVERT
